DOCS.md

```diff
@@ -86,7 +86,11 @@ Using a RegExp literal, the same snippet can be written as
 
 
 ### Snippet variables
-Snippet variables are used as shortcuts when writing snippets. By default, the following variables are available for use in a `trigger`:
+Snippet variables are used as shortcuts when writing snippets. Strings with the format `"${VARIABLE}"` will be replaced with the contents of  `VARIABLE` if it's defined (and left unchanged otherwise).
+
+Snippet variables can be changed in the settings, under **Advanced editor settings > Snippet variables**. They are defined using a JavaScript object. For your convince `VARIABLE`, `"VARIABLE"` and `"${VARIABLE}"` are all accepted.
```

Comment:
```suggestion
Snippet variables are used as shortcuts when writing snippets. In the `trigger` of a snippet, strings with the format `"${VARIABLE}"` will be replaced with the contents of the matching variable if it's defined, and left unchanged otherwise.

Snippet variables can be changed in the settings, under **Advanced editor settings > Snippet variables**. They are defined using a JavaScript object.
```